src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportTaskMetricsNotification.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.Core.Features.Metrics;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.Models;
+using Microsoft.Health.Fhir.ValueSets;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Export
+{
+    public class ExportTaskMetricsNotification : IMetricsNotification
+    {
+        public ExportTaskMetricsNotification(ExportJobRecord exportJobRecord)
+        {
+            FhirOperation = AuditEventSubType.Export;
+            ResourceType = null;
+
+            ExportJobRecord = exportJobRecord;
+        }
+
+        public string FhirOperation { get; }
+
+        public string ResourceType { get; }
+
+        public ExportJobRecord ExportJobRecord { get; }
```

Comment:
Do we really want to publish the entire `ExportJobRecord` object as part of the notification? It contains a lot of information that are completely unnecessary for metrics. I think it might be better to include only the required information.

Reply:
Thanks, I updated to use only few metrics from record.